packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -20,8 +20,24 @@ const StyledHeading = styled(Heading)`
   text-shadow: 0 2px 4px rgba(0,0,0,0.5);
 `
 
+const StyledAnchor = styled(Anchor)`
+  border-bottom: 3px solid transparent;
+  &:focus,
+  &:hover {
+    text-decoration: none;
+    border-color: white;
+  }
+`
+
 function ProjectHeader (props) {
-  const { className, title } = props
+  const { className, href, title } = props
+
+  const Title = () => (
+    <StyledHeading color='white' margin='none' size='small'>
+      {title}
+    </StyledHeading>
+  )
+
   return (
     <StyledBox>
```

Comment:
`className` needs to be set passed on to the outer component here to avoid class mismatch errors.